ubuntu/R/README.md

```diff
@@ -0,0 +1,21 @@
+# R container
+
+This image shows how to setup R and RStudio Server. 
+
+Note that you will still need to use an init script to start the RStudio Server
+daemon. Here is an example notebook python cell that installs an init script
+on a DBFS location.
+
+```py
+script = """#!/bin/bash
+set -euxo pipefail
+RSTUDIO_BIN="/usr/sbin/rstudio-server"
+
+if [[ ! -f "$RSTUDIO_BIN" && $DB_IS_DRIVER = "TRUE" ]]; then
+  rstudio-server restart || true
+fi
+"""
+
+dbutils.fs.mkdirs("/databricks/rstudio")
+dbutils.fs.put("/databricks/rstudio/rstudio-install.sh", script, True)
```

Comment:
oh please don't document more v1 init scripts. Point them to the init script docs instead.

Reply:
What are v1 init scripts? 

Im not sure how pointing to docs works here. Because the links will be different for azure and aws.

Reply:
updated the readme. Now we dont talk about configuring the init script